INSTALL.md

```diff
@@ -105,6 +105,81 @@ For Example:
 ./tools/tpm2_getrandom 4
 ```
 
+### Build example
+
+#### Fedora 27
+
+Here we are going to build tpm2-tools with:
+* tpm2-tss: <https://github.com/tpm2-software/tpm2-tss>
+* tpm2-abrmd: <https://github.com/tpm2-software/tpm2-abrmd>
+* TPM simulator: <https://downloads.sourceforge.net/project/ibmswtpm2/ibmtpm974.tar.gz>
+
+First we need to satisfy the dependencies for each item named above except for the simulator, in the next code block we install a mix of packages that should get the work done.
+
+
+```
+$ sudo dnf -y update && sudo dnf -y install automake libtool \
+autoconf autoconf-archive libstdc++-devel gcc pkg-config \
+uriparser-devel libgcrypt-devel dbus-devel glib2-devel \
+compat-openssl10-devel libcurl-devel PyYAML
+
+```
+For more detailed information about the dependencies of tpm2-tss and tmp2-abrmd, please consult the links provided above.
+
+Now we can start building the packages  
+
+* ##### Tpm2-tss
+```
+$ git clone git://github.com/tpm2-software/tpm2-tss.git
+$ cd tpm2-tss
+$ ./bootstrap
+$ ./configure --prefix=/usr
+$ make -j5
+```
+
+* ##### Tpm2-abrmd
+```
+$ git://github.com/tpm2-software/tpm2-abrmd.git
+$ cd tpm2-abrmd
+$ ./bootstrap
+$ ./configure --with-dbuspolicydir=/etc/dbus-1/system.d
+--with-udevrulesdir=/usr/lib/udev/rules.d
+--with-systemdsystemunitdir=/usr/lib/systemd/system
+--libdir=/usr/lib64 --prefix=/usr
+$ make -j5
+```
+
+* ##### Tpm2-tools
+```
+$ git clone git://github.com/tpm2-software/tpm2-tools.git
+$ cd tpm2-tools
+$ ./bootstrap
+$ ./configure --prefix=/usr
+$ make -j5
+```
+* ##### TPM simulator
+```
+$ mkdir ibmtpm && cd ibmtpm
+$ wget https://downloads.sourceforge.net/project/ibmswtpm2/ibmtpm974.tar.gz
+$ tar -zxvf ibmtpm974.tar.gz
+$ cd src
+$ make -j5
+```
+Depending of the version of fedora you are running you might have problems with the make command:
+![TPM simulator issue](/fedora-shared/tpm_simulator.png)
+This issue is present on fedora 26/27, it could be an issue with an older toolchain on the simulator than the one used by fedora 26/27, also, the -Werror wont let any build warning pass, so we need to patch this.
+The patch we are going to use is this: https://hastebin.com/raw/ovayaxamim
```

Comment:
I believe the links in hastebin are temporary, so this will become a broken link as well. But as mentioned, better to post that patch to the TPM simulator developers.

Reply:
This would posting the patch to ibmtpm20tss-users@lists.sourceforge.net I think.